python/ql/lib/semmle/python/dataflow/new/internal/LocalSources.qll

```diff
@@ -151,7 +153,7 @@ class LocalSourceNode extends Node {
    * See `TypeBackTracker` for more details about how to use this.
    */
   pragma[inline]
-  LocalSourceNode backtrack(TypeBackTracker t2, TypeBackTracker t) { t2 = t.step(result, this) }
+  LocalSourceNode backtrack(TypeBackTracker t2, TypeBackTracker t) { t = t2.step(result, this) }
```

Comment:
I don't think the QL doc changed here?

Reply:
No, and even if it had mentioned `t` and `t2` it would not have to change. It just makes it clear that there are two possible conventions here and we have two implementations that made different choices. So we should probably make explicit what role `t` and `t2` are supposed to play. It is tangential to this PR, though.